dangerzone/isolation_provider/container.py

```diff
@@ -154,7 +154,7 @@ def exec(
             args,
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
-            stderr=self.proc_stderr,
+            stderr=subprocess.PIPE,
```

Comment:
I think we need to reinstate the `self.proc_stderr` value, else we will always log to a pipe, even if we don't capture stderr.